### PR TITLE
update image markdown link to make it show in website

### PR DESCRIPTION
**Pull request by athom**
the image is not show up in webpage https://docs.fluentd.org/buffer
This PR try to fix it

image

### CODE REVIEW

**Comment by ganmacs** [comment]
Thank you! looks good! 
Could you add your sign to commit? https://github.com/fluent/fluentd-docs-gitbook/pull/53/checks?check_run_id=157746282

File: plugins/buffer/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ gets transferred to the destination).
 
 <div>
 
-[![](/images/fluentd-v0.14-plugin-api-overview.png)](/images/fluentd-v0.14-plugin-api-overview.png)
+[![](/images/fluentd-v0.14-plugin-api-overview.png)](https://github.com/fluent/fluentd-docs-gitbook/blob/1.0/images/fluentd-v0.14-plugin-api-overview.png)
```
Comment (ganmacs) [inline plugins/buffer/README.md]: Is`` correct?

Comment (athom) [reply plugins/buffer/README.md]: actually I don't understand why  this format `[x](y)`, 
can't this `x` work?

File: plugins/buffer/README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ gets transferred to the destination).
 
 <div>
```
Comment (ganmacs) [inline plugins/buffer/README.md]: I found how to fix it.  Removing this `<div>` tag and L46 `</div>` tag will fix.

File: plugins/buffer/README.md (hunk 1 above)
Comment (ganmacs) [reply plugins/buffer/README.md]: `[x](y)` means an image with a link like `<a href="/path1"></a>`.

File: plugins/buffer/README.md (hunk 2 above)
Comment (athom) [reply plugins/buffer/README.md]: ah, thanks for pointing out.
I updated the PR.

**Comment by ganmacs** [comment]
thank you!

**Pull request merged**